.gitignore

```diff
@@ -27,6 +27,12 @@
 /*.json
 install/gcp/*.json
 
+# Ignore JSON key for service account and not all JSON files.
+service-account-key.json
```

Comment:
Hmmm, why is this file in the repo in the first place?  Can we expect this file `service-account-key.json` to be placed outside of the git repo, rather than ignoring it inside the repo?

Reply:
Agree, these service account keys should be stored outside the repo.

Reply:
The best solution to implement is likely git-secrets checks: https://github.com/forseti-security/forseti-security/issues/3315

The internal docs on setting up an integration tests currently has you create this file in the repo directory. That needs to be updated with best practice on keeping this out of the repo path.In addition the file travis-integration.env likely needs to follow this same pattern.